tutor/src/models/user/terms.js

```diff
@@ -30,6 +30,7 @@ class UserTerms extends BaseModel {
 
   onLoaded({ data }) {
     this.terms = data;
+    this.user.terms_signatures_needed = this.terms.length > 0;
```

Comment:
I think this should look at `this.unsigned` instead of `this.terms`.

Reply:
That way it would match the BE's behavior: https://github.com/openstax/tutor-server/blob/master/app/representers/api/v1/user_representer.rb#L74